src/o365/spo/commands.ts

```diff
@@ -35,6 +35,7 @@ export default {
   SITE_APPCATALOG_REMOVE: `${prefix} site appcatalog remove`,
   SITE_GET: `${prefix} site get`,
   SITE_LIST: `${prefix} site list`,
+  SITE_LIST_CLASSIC: `${prefix} site list classic`,
```

Comment:
When we name commands, we put the verb add the end, so `site classic list`, instead of `site list class`. See also the command spec #122